tendrl/alerting/definitions/alerting.py

```diff
@@ -0,0 +1,47 @@
+# flake8: noqa
```

Comment:
Name of the file should be something like tendrl_definitions_alerting.py.
Also the package name followed in other cases are like tendrl/{module}/manager. Can we think of something similar?